themes/README.rst

```diff
@@ -0,0 +1,146 @@
+#####################
```

Comment:
This will not be the final place for docs.  I wanted a place to write instructions.  I'll work with the doc team to find the right place for this text when it is ready.